coderadar-test/src/test/java/io/reflectoring/coderadar/rest/query/GetCommitsInProjectControllerTest.java

```diff
@@ -74,6 +74,27 @@ void returnsAllCommitsInProject() throws Exception {
     Assertions.assertEquals("modify testModule1/NewRandomFile.java", commits.get(0).getComment());
   }
 
+  @Test
+  void returnsAllCommitsInProjectForContributor() throws Exception {
```

Comment:
Document this test (request + response) and include it in the `restapi.adoc`-file.

Reply:
Done